### PR TITLE
make gz calls idemptotent

### PR DESCRIPTION
**Pull request by coffeepac**
force all calls to gzip to happen an arbitrary, fixed time

### CODE REVIEW

File: ansible/roles/kraken.config/filter_plugins/gz.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import StringIO
 import gzip
+import time
```
Comment (davidewatson) [inline ansible/roles/kraken.config/filter_plugins/gz.py]: Are you using time?

**Comment by davidewatson** [comment]
LGTM, modulo my one question.

File: ansible/roles/kraken.config/filter_plugins/gz.py (hunk 1 above)
Comment (coffeepac) [inline ansible/roles/kraken.config/filter_plugins/gz.py]: nope.  thought I was going to need it for setting a consistent time for the gzip but I just hardcoded it instead.  I'll remove it, good catch.

**Comment by coffeepac** [comment]
fixes #118

**Pull request merged**